core/ui.libs/src/style/definitions/evan.theme.scss

```diff
@@ -239,15 +239,14 @@ $themeEvan: map-merge(
  */
 $themeEvan: map-merge(
   (
-    h1-font-size:    2.250em,   // 36px / 2.250em / 2.250rem
-    h2-font-size:    1.750em,   // 28px / 1.750em / 1.750rem
-    h3-font-size:    21px   ,   // 21px / 1.313em / 1.313rem
-    h4-font-size:    18px   ,   // 18px / 1.125em / 1.125rem
-    h5-font-size:    16px   ,   // 16px / 1em     / 1rem
-    h6-font-size:    15px   ,   // 15px / 0.938em / 0.938rem
-    // text-font-size:  14px   ,   // 14px / 0.875em
-    text-font-size:  12px   ,   // 14px / 0.875em
-    small-font-size: 10px   ,
+    h1-font-size:    1.5rem, // 18px for all
+    h2-font-size:    1.5rem,
+    h3-font-size:    1.5rem,
+    h4-font-size:    1.5rem,
+    h5-font-size:    1.5rem,
+    h6-font-size:    1.5rem,
```

Comment:
side effects incooooooming :D

Reply:
The new concept doesn't discriminate between heading levels, so there shouldn't be an issue (hopefully :D)

Reply:
The current concept not, but we have several. custom styles so side effects are highly expected. We should also refactore all the headlines then and maybe get rid of h6, h5, h4, ...

Reply:
So it's a good start to catch those headings, I guess? ;) 
According to Daniel, all headings should be same size. If anything behaves badly now, it shouldn't have been a heading in the first place.

Reply:
I don't disagree, it's just a tough choice to do it now in this PR ;)

Reply:
I only noticed "improvements" in the form of uniform headings across the product so far. I'll revert/fix it if anything goes wrong ;)

Reply:
According Daniel, yes ;)